drivers/scheduler/k8s/specs/elasticsearch/px-elasticdata-app.yaml

```diff
@@ -119,3 +131,29 @@ spec:
       resources:
         requests:
           storage: 5Gi
+---
+apiVersion: apps/v1
+kind: ReplicaSet
+metadata:
+  name: elasticrearch-load
+  namespace: default
+  labels:
+    app: esrally
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: esrally
+  template:
+    metadata:
+      labels:
+        app: esrally
+    spec:
+      containers:
+      - name: esrally
+        image: elastic/rally:1.3.0
+        command:
+          - /bin/sh
+          - -c
+          - "while true; do esrally --pipeline=benchmark-only --target-hosts=esnode-0.elasticsearch-cluster.default.svc.cluster.local:9200,esnode-1.elasticsearch-cluster.default.svc.cluster.local:9200,esnode-2.elasticsearch-cluster.default.svc.cluster.local:9200; sleep 5; done"
```

Comment:
We need to parameterize the load and time that it runs for . Can you see how can we do that ?

Reply:
From torpedo standpoint, it might be okay if it keeps running through the course of the test.

Only concern would be the rate of IO this generates. Do we know the throughput or IOPS the above command would generate?

Reply:
I haven't found any time parameters for track/tests. Try to find an alternative tool for this.